industry_benchmarks/input_structures/prepared_structures/charge_annihilation_set/thrombin/PREPARATION_DETAILS.md

```diff
@@ -0,0 +1,11 @@
+## System preparation details
+
+_For example:_
+
+* All prepration was carried out using maestro and text editors.
+* Extracted GOL cofactor.
+* Removed PTM NAG and saturated the connected ASN.
+* Converted TYS to TYR.
+* Converted N-terminal H atom to H3 for 1C GLU (chain L), same for other chains.
+* Converted C-terminal O atom to OXT for 14K ILE (chain L), same for other chains.
+* Removed ligand
```

Comment:
Could you please add further details here about which ligand was removed? Thanks!

Reply:
My bad, no ligand removed for this one